slurm_main.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash -l
```

Comment:
why do you delete this file?

Reply:
feel rit computing resource is not in need at this moment. want to clean up our root files. Will get them back in a later PR after code refactoring